### DISCUSSION TITLE
Sort by total time and timecremaining

### DISCUSSION THREAD

**Discussion by jrm72** [PaulWoitaschek/Voice#2564] [Ideas]
If you add a sort function, could it include sort by total time and also time remaining.

Kindest regards

James ❤️🖖

**Comment by Marloni21** [comment]
O aplicativo é excelente, seria melhor ainda se pudesse adicionar automaticamente as  palavras do audiolivro no marcador de páginas e tivesse backups online, assim ficaria mais prático e poderia guardar o resumo do audiolivro na nuvem.
